CRM/Utils/System.php

```diff
@@ -24,6 +24,7 @@
  *
  * @method static void getCMSPermissionsUrlParams() Immediately stop script execution and display a 401 "Access Denied" page.
  * @method static mixed permissionDenied() Show access denied screen.
+ * @method static string getContentTemplate(bool $print) Get the template path to render whole content.
```

Comment:
add the "frontEnd" param here?

Reply:
~add the "frontEnd" param here?~

remove the "frontEnd" param everywhere else?